pkgs/applications/networking/browsers/chromium/ungoogled.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv
+, fetchFromGitHub
+, python3Packages
+, makeWrapper
+, patch
+}:
+{ rev, sha256 }:
+stdenv.mkDerivation {
```

Comment:
This expression lacks good whitespace like other expressions in nixpkgs.

See like https://github.com/NixOS/nixpkgs/blob/master/pkgs/desktops/pantheon/apps/elementary-camera/default.nix

Reply:
Updated whitespace.